components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -1425,28 +1420,32 @@ std::vector<mojom::NetworkInfoPtr> GetAllKnownChains(PrefService* prefs,
 
   if (coin == mojom::CoinType::SOL) {
     for (const auto* network : GetKnownSolNetworks()) {
-      result.push_back(network->Clone());
+      result.push_back(
+          GetKnownChain(prefs, network->chain_id, mojom::CoinType::SOL));
```

Comment:
```
result.push_back(network->Clone());
```
Should be back here and below

Reply:
https://github.com/brave/brave-core/pull/24045/commits/50cc22db2726e9b9430b03ea7a1e2ca96727c157